Makefile

```diff
@@ -20,3 +20,6 @@ migrate: ## Run migrations
 
 create-empty-migration: ## Create an empty migration 
 	docker-compose exec api rails g migration
+
+create-super-user: ## Create a test super user with creds testuser@now-u.com, strongpassword
```

Comment:
Can we rather put db seeds in the seeds file (`seeds.rb`) and populate it using Factory Bot (`spec/factories/admins.rb`) and then replace this command with the db seed command